index.html

```diff
@@ -0,0 +1,139 @@
+                <h3 class="product-name">Pizza Buratta</h3>
+                <h4 class="product-category">Pizza</h4>
+                <h4 class="product-price">11 $</h4>
+            </div>
+            <div class="individual-product">
+                <img id="product2" src="https://www.allrecipes.com/thmb/5SdUVhHTMs-rta5sOblJESXThEE=/1500x0/filters:no_upscale():max_bytes(150000):strip_icc()/11691-tomato-and-garlic-pasta-ddmfs-3x4-1-bf607984a23541f4ad936b33b22c9074.jpg" alt="product2">
+                <h3 class="product-name">Pasta Bolognesse</h3>
+                <h4 class="product-category">Pasta</h4>
+                <h4 class="product-price">9.5 $</h4>
+            </div>
+            <div class="individual-product">
+                <img id="product3" src="https://demuths.co.uk/img/u/images/_256x256_crop_center-center_70_none/a0bbcdaf-8b01-4a10-99f2-cde9785d5e1d.jpeg" alt="product3">
+                <h3 class="product-name">Demuths Soup</h3>
+                <h4 class="product-category">Soup</h4>
+                <h4 class="product-price">8.5 $</h4>
+            </div>
+            <div class="individual-product">
+                <img id="product4" src="https://files.yappe.in/place/small/malabar-cafe-2538191.webp" alt="product4">
+                <h3 class="product-name">Yappe Soup</h3>
+                <h4 class="product-category">Soup</h4>
+                <h4 class="product-price">9.2 $</h4>
+            </div>
+            <div class="individual-product">
+                <img id="product5" src="https://www.seriouseats.com/thmb/Fi_FEyVa3_-_uzfXh6OdLrzal2M=/1500x0/filters:no_upscale():max_bytes(150000):strip_icc()/the-best-caesar-salad-recipe-06-40e70f549ba2489db09355abd62f79a9.jpg" alt="product5">
+                <h3 class="product-name">Caesar Salad</h3>
+                <h4 class="product-category">Salad</h4>
+                <h4 class="product-price">6.3 $</h4>
+            </div>
+        </div>
+
+        <br>
+        <br>
+        <br>
+        <br>
+    </body>
+
+<!-- footer -->
+    <footer>
+        <div id="footer">
+            <div id="footer-text">
+                <p id="footer-text-p">© 2021 Fire delight. All rights reserved.</p>
+            </div>
+            <div id="footer-social">
+                <a href="https://www.facebook.com/"><img src="https://www.facebook.com/images/fb_icon_325x325.png" alt="facebook" style="width:50px;height:50px;"></a>
+                <a href="https://www.instagram.com/"><img src="https://upload.wikimedia.org/wikipedia/commons/9/95/Instagram_logo_2022.svg" alt="instagram" style="width:50px;height:50px;"></a>
+                <a href="https://www.twitter.com/"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6f/Logo_of_Twitter.svg/1200px-Logo_of_Twitter.svg.png" alt="twitter" style="width:50px;height:50px;"></a>
+            </div>
+        </div>
+    </footer>
+</html>
```

Comment:
It's good practice to have newlines at the end of files. There is a VSCode setting to do that automatically on save. Otherwise, you can see this icon on Pull Requests:
![image](https://github.com/LaurentiuTusa/WP-week-1/assets/19929642/689bbc83-1506-4594-b1b8-f5e2420a4116)